signup.html

```diff
@@ -35,62 +27,77 @@ <h1>Signup Form</h1>
+          document.getElementById("name").classList.add("error-field");
+          isValid = false;
+        }
 
-          // Reset error messages and field styles
+        // Validation for Email
+        const email = document.getElementById("email").value;
+        const emailPattern = /\S+@\S+\.\S+/;
+        if (!emailPattern.test(email)) {
+          document.getElementById("emailError").textContent =
+            "Invalid email address.";
+          document.getElementById("email").classList.add("error-field");
+          isValid = false;
+        }
+
+        // Validation for Password (length must be at least 8 characters)
+        const password = document.getElementById("password").value;
+        if (password.length < 8) {
+          document.getElementById("passwordError").textContent =
+            "Password must be at least 8 characters.";
+          document.getElementById("password").classList.add("error-field");
+          isValid = false;
+        }
+
+        if (isValid) {
+          // Clear error messages and field styles
           document.getElementById("nameError").textContent = "";
           document.getElementById("name").classList.remove("error-field");
           document.getElementById("emailError").textContent = "";
           document.getElementById("email").classList.remove("error-field");
           document.getElementById("passwordError").textContent = "";
           document.getElementById("password").classList.remove("error-field");
 
-          // Validation for Name
-          const name = document.getElementById("name").value;
-          if (name.trim() === "") {
-            document.getElementById("nameError").textContent =
-              "Name is required.";
-            document.getElementById("name").classList.add("error-field");
-            isValid = false;
-          }
-
-          // Validation for Email
-          const email = document.getElementById("email").value;
-          const emailPattern = /\S+@\S+\.\S+/;
-          if (!emailPattern.test(email)) {
-            document.getElementById("emailError").textContent =
-              "Invalid email address.";
-            document.getElementById("email").classList.add("error-field");
-            isValid = false;
-          }
-
-          // Validation for Password (length must be at least 8 characters)
-          const password = document.getElementById("password").value;
-          if (password.length < 8) {
-            document.getElementById("passwordError").textContent =
-              "Password must be at least 8 characters.";
-            document.getElementById("password").classList.add("error-field");
-            isValid = false;
-          }
-
-          if (!isValid) {
-            event.preventDefault(); // Prevent form submission
-          }
-        };
-      </script>
-    </section>
+          // Display success message
+          const successMessage = document.getElementById("successMessage");
```

Comment:
This is the right way to use **getElementById**. Please fix the comments according this implementation.

Reply:
fix